<ui version="4.0" stdsetdef="1" >
  <author>Sebastian Trueg</author>
  <comment>Widget to edit the global CD-Text in a K3b audio project.</comment>
  <exportmacro></exportmacro>
  <class>base_K3bAudioCdTextWidget</class>
  <widget class="QWidget" name="K3bAudioCDTextWidget" >
    <property name="windowTitle" >
      <string>K3bAudioCDTextWidget</string>
    </property>
    <layout class="QVBoxLayout" >
      <item>
        <widget class="QGroupBox" name="m_groupCdText" >
          <property name="title" >
            <string>Write CD-Text</string>
          </property>
          <property name="checkable" >
            <bool>true</bool>
          </property>
          <property name="checked" >
            <bool>false</bool>
          </property>
          <property name="toolTip" stdset="0" >
            <string/>
          </property>
          <property name="whatsThis" stdset="0" >
            <string>&lt;p>&lt;b>CD-Text&lt;/b>
&lt;p>If this option is checked K3b uses some otherwise unused space on the Audio CD to store additional information, such as the artist's name or the CD title.
&lt;p>CD-Text is an extension to the audio CD standard introduced by Sony.
&lt;p>CD-Text will only be usable on CD players that support this extension (mostly car CD players) and software like K3b, of course.
&lt;p>Since a CD-Text-enhanced Audio CD will work in any Hifi CD or DVD player even if the player does not support CD-Text explicitly, enabling it is never a bad idea (just remember to fill in the CD-Text information).</string>
          </property>
          <layout class="QVBoxLayout" >
            <item>
              <widget class="QFrame" name="frame3" >
                <property name="frameShape" >
                  <enum>NoFrame</enum>
                </property>
                <property name="frameShadow" >
                  <enum>Raised</enum>
                </property>
                <layout class="QGridLayout" >
                  <property name="margin" >
                    <number>0</number>
                  </property>
                  <item row="2" column="0" >
                    <widget class="QLabel" name="textLabel2" >
                      <property name="text" >
                        <string>Perf&amp;ormer:</string>
                      </property>
                      <property name="buddy" stdset="0" >
                        <cstring>m_editPerformer</cstring>
                      </property>
                      <property name="wordWrap" >
                        <bool>false</bool>
                      </property>
                    </widget>
                  </item>
                  <item row="2" column="1" >
                    <layout class="QHBoxLayout" >
                      <item>
                        <widget class="KLineEdit" name="m_editPerformer" />
                      </item>
                      <item>
                        <widget class="QToolButton" name="m_buttonCopyPerformer" >
                          <property name="text" >
                            <string>...</string>
                          </property>
                          <property name="toolTip" stdset="0" >
                            <string>Copy to all tracks</string>
                          </property>
                        </widget>
                      </item>
                    </layout>
                  </item>
                  <item row="1" column="1" >
                    <layout class="QHBoxLayout" >
                      <item>
                        <widget class="KLineEdit" name="m_editTitle" />
                      </item>
                      <item>
                        <widget class="QToolButton" name="m_buttonCopyTitle" >
                          <property name="text" >
                            <string>...</string>
                          </property>
                          <property name="toolTip" stdset="0" >
                            <string>Copy to all tracks</string>
                          </property>
                        </widget>
                      </item>
                    </layout>
                  </item>
                  <item row="1" column="0" >
                    <widget class="QLabel" name="textLabel1" >
                      <property name="text" >
                        <string>&amp;Title:</string>
                      </property>
                      <property name="buddy" stdset="0" >
                        <cstring>m_editTitle</cstring>
                      </property>
                      <property name="wordWrap" >
                        <bool>false</bool>
                      </property>
                    </widget>
                  </item>
                </layout>
              </widget>
            </item>
            <item>
              <layout class="QHBoxLayout" >
                <item>
                  <spacer name="spacer3" >
                    <property name="sizeHint" >
                      <size>
                        <width>0</width>
                        <height>0</height>
                      </size>
                    </property>
                    <property name="sizeType" >
                      <enum>QSizePolicy::Expanding</enum>
                    </property>
                    <property name="orientation" >
                      <enum>Qt::Horizontal</enum>
                    </property>
                  </spacer>
                </item>
                <item>
                  <widget class="QPushButton" name="m_buttonMoreFields" >
                    <property name="text" >
                      <string>More Fiel&amp;ds...</string>
                    </property>
                  </widget>
                </item>
              </layout>
            </item>
            <item>
              <spacer name="spacer1" >
                <property name="sizeHint" >
                  <size>
                    <width>21</width>
                    <height>0</height>
                  </size>
                </property>
                <property name="sizeType" >
                  <enum>QSizePolicy::Expanding</enum>
                </property>
                <property name="orientation" >
                  <enum>Qt::Vertical</enum>
                </property>
              </spacer>
            </item>
          </layout>
        </widget>
      </item>
    </layout>
  </widget>
  <tabstops>
    <tabstop>m_editTitle</tabstop>
    <tabstop>m_editPerformer</tabstop>
  </tabstops>
  <includes>
    <include location="local" >klineedit.h</include>
  </includes>
</ui>
